### PR TITLE
doc: changelog fixes

### PR DESCRIPTION
**Pull request by divipillai**
changelog fixes

### CODE REVIEW

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -158,33 +158,26 @@ This section provides detailed lists of changes by :ref:`application <applicatio
...
 
+* Added the integration of the :ref:`lib_lwm2m_client_utils` FOTA callback functionality.
+
 * Updated:
 
-  * Use defines from the :ref:`lib_nrf_cloud` library for nRF Cloud related string values.
-  * The application now integrates the :ref:`lib_lwm2m_client_utils` FOTA callback functionality.
-  * The application now uses the function :c:func:`nrf_cloud_location_request_msg_json_encode` to create an nRF Cloud location request message.
+  * The application to use the function :c:func:`nrf_cloud_location_request_msg_json_encode` to create an nRF Cloud location request message.
+  * The application to use defines from the :ref:`lib_nrf_cloud` library for nRF Cloud-related string values.
```
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-changelog.rst:166]:
```suggestion
  * The application now uses the function :c:func:`nrf_cloud_location_request_msg_json_encode` to create an nRF Cloud location request message.
  * The application now uses defines from the :ref:`lib_nrf_cloud` library for nRF Cloud-related string values.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -199,16 +192,17 @@ nRF Desktop
 
 Added:
 
-* The :ref:`nrf_desktop_swift_pair_app`. The module is used to enable or disable the Swift Pair Bluetooth advertising payload depending on the selected Bluetooth peer (used local identity).
+* The :ref:`nrf_desktop_swift_pair_app` module.
```
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-changelog.rst:195]: Module name already has 'module'.
```suggestion
* The :ref:`nrf_desktop_swift_pair_app` .
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -251,16 +245,12 @@ nRF9160 samples
 
 * :ref:`modem_shell_application` sample:
 
-  * Updated:
-
-    * Use defines from the :ref:`lib_nrf_cloud` library for nRF Cloud related string values.
-      Remove the inclusion of the file :file:`nrf_cloud_codec.h`.
+  * Updated the sample to use defines from the :ref:`lib_nrf_cloud` library for nRF Cloud related string values.
+    Remove the inclusion of the file :file:`nrf_cloud_codec.h`.
```
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-changelog.rst:249]:
```suggestion
    Removed the inclusion of the file :file:`nrf_cloud_codec.h`.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 4]
```diff
@@ -158,33 +158,26 @@ This section provides detailed lists of changes by :ref:`application <applicatio
...
-* Updated:
-
-  * Use defines from the :ref:`lib_nrf_cloud` library for nRF Cloud related string values.
-
-* Fixed:
-
-  * A bug in receiving large MQTT Publish message.
+* Added AT command ``#XWIFIPOS`` to get Wi-Fi location from nRF Cloud.
+* Updated the application to use defines from the :ref:`lib_nrf_cloud` library for nRF Cloud related string values.
+* Fixed a bug in receiving large MQTT Publish message.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:173]:
```suggestion
* Fixed a bug in receiving a large MQTT Publish message.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 5]
```diff
@@ -251,16 +245,12 @@ nRF9160 samples
...
-      Remove the inclusion of the file :file:`nrf_cloud_codec.h`.
+  * Updated the sample to use defines from the :ref:`lib_nrf_cloud` library for nRF Cloud related string values.
+    Removed the inclusion of the file :file:`nrf_cloud_codec.h`.
 
 * :ref:`https_client` sample:
 
-  * Added:
-
-    * Add IPv6 support and wait for PDN to fully activate (including IPv6, if available) before looking up address.
+  * Added IPv6 support and wait for PDN to fully activate (including IPv6, if available) before looking up the address.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:253]: This sounds a bit weird. "Added IPv6 support and wait for..."
However, I don't have a good suggestion to make this better.

File: doc/nrf/releases/release-notes-changelog.rst [hunk 6]
```diff
@@ -158,33 +158,26 @@ This section provides detailed lists of changes by :ref:`application <applicatio
...
 -------------------------
 
+* Added the integration of the :ref:`lib_lwm2m_client_utils` FOTA callback functionality.
+
 * Updated:
 
-  * Use defines from the :ref:`lib_nrf_cloud` library for nRF Cloud related string values.
-  * The application now integrates the :ref:`lib_lwm2m_client_utils` FOTA callback functionality.
   * The application now uses the function :c:func:`nrf_cloud_location_request_msg_json_encode` to create an nRF Cloud location request message.
+  * The application now uses defines from the :ref:`lib_nrf_cloud` library for nRF Cloud-related string values.
```
[the remark this answers is not in the document]
Comment (divipillai) [reply doc/nrf/releases/release-notes-changelog.rst]: I think second option is better. I have updated in all the three places.

File: doc/nrf/releases/release-notes-changelog.rst (hunk 5 above)
Comment (divipillai) [reply doc/nrf/releases/release-notes-changelog.rst]: I updated the sentence. Please check.

**Review by peknis — APPROVED** [review]

**Review by umapraseeda — APPROVED** [review]
Line433: typo -> Standardized

**Pull request merged**